core/services/bulletprooftxmanager/eth_broadcaster.go

```diff
@@ -472,7 +449,7 @@ func (eb *ethBroadcaster) tryAgainWithHigherGasPrice(sendError *eth.SendError, e
 	if bumpedGasPrice.Cmp(attempt.GasPrice.ToInt()) == 0 && bumpedGasPrice.Cmp(eb.config.EthMaxGasPriceWei()) == 0 {
 		return errors.Errorf("Hit gas price bump ceiling, will not bump further. This is a terminal error")
 	}
-	replacementAttempt, err := newAttempt(eb.store, etx, bumpedGasPrice)
+	replacementAttempt, err := newAttempt(context.TODO(), eb.store, etx, bumpedGasPrice)
```

Comment:
Can we use the `eth.DefaultQueryCtx` here instead? I'm nervous since I know the websocket can permahang sometimes and this is a critical path.